.gitignore

```diff
@@ -1,4 +1,4 @@
 dcos-signal
-signal_*
+./signal_*
```

Comment:
Why do we need it?

Reply:
This is prevents the binary that gets created from accidentally getting added to git